lib/ui/views/explore/explore_page_view.dart

```diff
@@ -144,6 +144,7 @@ class _ExploreTabsState extends State<ExploreTabs>
         backgroundColor: Theme.of(context).colorScheme.background,
         elevation: 1,
         shadowColor: CustomColors.greyLight1,
+		automaticallyImplyLeading: false,
```

Comment:
```suggestion
	automaticallyImplyLeading: false,
```
Isn't the spacing wrong here?